src/Microsoft.DotNet.Web.ProjectTemplates/content/EmptyWeb-CSharp/.template.config/template.json

```diff
@@ -110,7 +110,7 @@
     "copyrightYear": {
       "type": "generated",
       "generator": "now",
-      "replaces": "1975",
+      "replaces": "copyrightYear",
```

Comment:
We already had the macro lined up. Just updated to a better variable to replace.